README.md

```diff
@@ -102,6 +102,11 @@ An object containing the following properties which provide all the data-specifi
 
 These control the size of the viewport in your app.
 
+##### `overviewOn` (boolean) [ONLY NECESSARY FOR `VivViewer`]
+
+This is a controller for whether or not you have an overview (picture-in-picture).  This only works
+with `loader.isPyramid` set to `true`.
```

Comment:
well yes, but also loader needs to implement `getRasterSize` now as well, no?

Reply:
Ok, I can add that in.